src/Task/ListItem.ts

```diff
@@ -16,4 +16,22 @@ export class ListItem {
             parent.children.push(this);
         }
     }
+
+    /**
+     * Parses the parents recursively until a {@link ListItem} with a `null` parent is found.
+     */
```

Comment:
A general tip about writing docs is to focus on things like:

- why someone might need to call it
- what its behaviour is
- what any non-obvious aspects of the behaviour are

And to avoid restating the implementation.

I will push some edits to this, for expediency - you can decide if they are any better...

Reply:
Things to mention might be that the root of a top-level (unindented) task is itself.